### PR TITLE
Splitter: describe API

### PR DESCRIPTION
**Pull request by vladaskorohodova**

### CODE REVIEW

File: api-reference/10 UI Components/dxSplitter/7 Interfaces/dxSplitterItem/collapsedSize.md [hunk 1]
```diff
@@ -5,7 +5,67 @@ default: undefined
...
+---
+##### jQuery
+
+    <!-- tab: index.js -->
+    $(() => {
+        $("#splitter").dxSplitter({
+            items: [
+                {
+                    // ...
+                    collapsedSize: "20",
```
Comment (EugeniyKiyashko) [inline api-reference/10 UI Components/dxSplitter/7 Interfaces/dxSplitterItem/collapsedSize.md]:
```suggestion
                    collapsedSize: "20px",
```

File: api-reference/10 UI Components/dxSplitter/7 Interfaces/dxSplitterItem/collapsedSize.md [hunk 2]
```diff
@@ -5,7 +5,67 @@ default: undefined
...
+            collapsedSize="20"
+        >
+        </dxi-item>
+    </dx-splitter>
+
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template>
+        <DxSplitter item-template="item">
```
Comment (EugeniyKiyashko) [inline api-reference/10 UI Components/dxSplitter/7 Interfaces/dxSplitterItem/collapsedSize.md]:
```suggestion
        <DxSplitter ...>
```

File: api-reference/10 UI Components/dxSplitter/7 Interfaces/dxSplitterItem/maxSize.md [hunk 3]
```diff
@@ -5,7 +5,65 @@ default: undefined
...
+---
+##### jQuery
+
+    <!-- tab: index.js -->
+    $(() => {
+        $("#splitter").dxSplitter({
+            items: [
+                {
+                    // ...
+                    maxSize: "500",
```
Comment (EugeniyKiyashko) [inline api-reference/10 UI Components/dxSplitter/7 Interfaces/dxSplitterItem/maxSize.md]:
```suggestion
                    maxSize: "50%",
```

File: api-reference/10 UI Components/dxSplitter/7 Interfaces/dxSplitterItem/splitter.md [hunk 4]
```diff
@@ -5,7 +5,118 @@ default: undefined
...
+                },
+                {
+                    splitter: {
+                        orientation: "vertical",
+                        items: [{
+                                text: "Pane 2"
+                            },
+                            {
+                                splitter: {
+                                    orientation: "horizontal",
```
Comment (EugeniyKiyashko) [inline api-reference/10 UI Components/dxSplitter/7 Interfaces/dxSplitterItem/splitter.md]: Orientation's default value is horizontal

File: api-reference/_hidden/CollectionWidgetItem/template.md [hunk 5]
```diff
@@ -126,26 +147,20 @@ The following example adds a custom item to the component. Note that Angular and
...
-                    </Item>
-                </{WidgetName}>
-            );
-        }
+    const App() = () => {
+        return (
+            <{WidgetName} ... >
+                <Item ...
+                    render={renderCustomItem}
+                >
```
Comment (EugeniyKiyashko) [inline api-reference/_hidden/CollectionWidgetItem/template.md:155]:
```suggestion
                />
```

File: api-reference/10 UI Components/dxSplitter/1 Configuration/itemTemplate.md [hunk 6]
```diff
@@ -0,0 +1,97 @@
...
+##### jQuery
+
+    <!-- tab: index.js -->
+    $(() => {
+        $("#splitter").dxSplitter({
+            items: [
+                // ...
+            ],
+            itemTemplate: function (data, index, element) {
+                let randomColor = "#" + Math.floor(Math.random()*16777215).toString(16);
```
Comment (EugeniyKiyashko) [inline api-reference/10 UI Components/dxSplitter/1 Configuration/itemTemplate.md:13]: const randomColor = `#${Math.floor(Math.random() * 0xFFFFFF).toString(16).padStart(6, '0')}`;

File: api-reference/10 UI Components/dxSplitter/7 Interfaces/dxSplitterItem/size.md [hunk 7]
```diff
@@ -5,7 +5,75 @@ default: undefined
...
+
+    export default App;
+
+---
+
+If you do not specify pane sizes, the UI component splits up the panes automatically with even distribution.
+
+[note]
+
+- The **size** value must not exceed [maxSize](/Documentation/ApiReference/UI_Components/dxSplitter/Interfaces/dxSplitterItem/#maxSize) and be less then [minSize](/Documentation/ApiReference/UI_Components/dxSplitter/Interfaces/dxSplitterItem/#minSize).
```
Comment (Abadzhev) [inline api-reference/10 UI Components/dxSplitter/7 Interfaces/dxSplitterItem/size.md]: I don't like the phrase "must not exceed". What will happen if I do try to assign a value that's out of bounds? Size will remain unchanged? Or maybe the size will be set to closest limit? 

If we don't want to answer that question, I guess it's better to still use imperative mood. "Use max and min to specify contstraints."

```suggestion
- You can use [minSize](/Documentation/ApiReference/UI_Components/dxSplitter/Interfaces/dxSplitterItem/#minSize) and [maxSize](/Documentation/ApiReference/UI_Components/dxSplitter/Interfaces/dxSplitterItem/#maxSize) properties to specify size constraints.
```

File: api-reference/10 UI Components/dxSplitter/1 Configuration/onResize.md [hunk 8]
```diff
@@ -5,25 +5,25 @@ default: null
...
-<!-- Description goes here -->
+A function that is executed each time an item (pane) is resized by one pixel.
 
 ##### param(e): ui/splitter:ResizeEvent
-<!-- Description goes here -->
+Information about the event.
 
 ##### field(e.cancel): Boolean
-<!-- Description goes here -->
+Allows you to cancel resize operation.
```
Comment (dirk-pieterse) [inline api-reference/10 UI Components/dxSplitter/1 Configuration/onResize.md]:
```suggestion
Allows you to cancel the resize operation.
```

File: api-reference/10 UI Components/dxSplitter/1 Configuration/onResizeEnd.md [hunk 9]
```diff
@@ -5,25 +5,25 @@ default: null
...
-<!-- Description goes here -->
+A function that is called when resizing ends.
 
 ##### param(e): ui/splitter:ResizeEndEvent
-<!-- Description goes here -->
+Information about the event.
 
 ##### field(e.cancel): Boolean
-<!-- Description goes here -->
+Allows you to cancel resize operation.
```
Comment (dirk-pieterse) [inline api-reference/10 UI Components/dxSplitter/1 Configuration/onResizeEnd.md]:
```suggestion
Allows you to cancel the resize operation.
```

**Review by dirk-pieterse — APPROVED** [review]

**Pull request merged**